image_load.py

```diff
@@ -0,0 +1,59 @@
+from pexelsapi.pexels import Pexels
+import requests
+import logging
+import random
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("Image_Load_Logger")
+
+
+class ImageException(Exception):
+    pass
+
+
+class APINotAvailable(ImageException):
+    pass
+
+
+class PexelAPIKeyError(ImageException):
+    pass
+
+
+def get_image_url(pexel_api_key):
+    try:
+        logger.info('Connecting to Pexels API...')
+        pexel = Pexels(pexel_api_key)
+        search_photos = pexel.curated_photos(page=1, per_page=80)
+        logger.info('Getting image url...')
+        image_url = search_photos['photos'][random.randint(0, 79)]['src']['tiny']
+        logger.info('Image url received')
+
+        return image_url
+
+    except requests.exceptions.JSONDecodeError as e:
+        logger.info("Pexel's API Unavailable. Can't receive image url")
+        logger.info(e)
+
+        raise APINotAvailable('Check the availability of Pexel API')
+
+    except KeyError as e:

```

Comment:
I believe `keyerror` can be only result of string 28, so you should split big try...except block into two blocks.

```python
    try:
        pexel = Pexels(pexel_api_key)
        search_photos = pexel.curated_photos(page=1, per_page=80)
        # bla bla bla
    except requests.exceptions.JSONDecodeError as e:
        # bla bla bla
        
    try:
        image_url = search_photos['photos'][random.randint(0, 79)]['src']['tiny']
    except KeyError as e:
        # bla bla bla
    else:
        logger.info('Image url received')
        return image_url
```

You can also use `dict.get()` method and check image_url for None and return it or raise an exception (its even better then try...except, IMO).